apps/ai-content-generator/src/components/common/text-counter/TextCounter.styles.ts

```diff
@@ -0,0 +1,13 @@
+import { css } from '@emotion/react';
+
+export const styles = {
+  // How do I access these colors from forma36?
```

Comment:
Is this a thing we can do?

Reply:
Yes, you can access the forma36 colors from tokens, here is an example: https://github.com/contentful/apps/blob/master/apps/ai-content-generator/src/components/app/dialog/common-generator/CommonGenerator.styles.tsx#L7